bindings/jupyter-modules/jupyter-manifold/package.json

```diff
@@ -26,6 +26,9 @@
     "reduce-reducers": "^1.0.4",
     "redux": "^4.0.0",
     "redux-actions": "^2.6.5",
-    "redux-thunk": "^2.3.0"
+    "redux-thunk": "^2.3.0",
+    "styled-components": "^4.3.2",
```

Comment:
@kenns29 @Firenze11 do we need `styled-components` here in the `jupyter-manifold` module?

Reply:
@gnavvy moved the styled-components into peer dependencies.